packages/notification-center/src/components/notification-center/components/layout/header/Header.tsx

```diff
@@ -7,8 +7,9 @@ import { INotificationCenterContext } from '../../../../../index';
 import { NotificationCenterContext } from '../../../../../store/notification-center.context';
 import { I18NContext } from '../../../../../store/i18n.context';
 import { Cogs } from '../../../../../shared/icons';
+import { Screens } from '../Layout';
 
-export function Header({ unseenCount, setShowSettings }: { unseenCount: number; setShowSettings: (boolean) => void }) {
+export function Header({ unseenCount, setScreen }: { unseenCount: number; setScreen: (Screens) => void }) {
```

Comment:
```suggestion
export function Header({ unseenCount, setScreen }: { unseenCount: number; setScreen: (screen: Screens) => void }) {
```

I think it will provide a better annotation for the type.